README.md

```diff
@@ -54,6 +54,22 @@ cfn test --enforce-timeout 60 #set the RL handler timeout to 60 seconds and CUD
 cfn test --enforce-timeout 60 -- -k contract_delete_update # combine args
 ```
 
+### Command: build-image
+
+To build an image. This image provides a minimalistic execution environment for the handler that does not depend on AWS Lambda in anyway. This image can be used during cfn invoke and cfn test instead of using sam cli.
+
+```bash
+cfn test
```

Comment:
Could we have example of this command itself? `cfn build-image ....`?

Reply:
Fixed